include/secp256k1_commitment.h

```diff
@@ -181,6 +181,29 @@ SECP256K1_API SECP256K1_WARN_UNUSED_RESULT int secp256k1_pedersen_blind_generato
   size_t n_inputs
 );
 
+/** Calculates the blinding factor x' = x + SHA256(xG+vH | xJ), used in the switch commitment x'G+vH
+ *
+ * Returns 1: Blinding factor successfully computed.
+ *         0: Error. Retry with different values.
+ *
+ * Args:           ctx: pointer to a context object
+ * Out:   blind_switch: blinding factor for the switch commitment
+ * In:           blind: pointer to a 32-byte blinding factor
+ *               value: unsigned 64-bit integer value to commit to
+ *           value_gen: value generator 'h'
+ *           blind_gen: blinding factor generator 'g'
+ *       switch_pubkey: pointer to public key 'j'
```

Comment:
How about `switch_gen` here?  because `J` is a Generator, an base point which is chosen to be orthogonal to G, , similar as `H`.

Since we use `value_gen`, `blind_gen`, so I think `switch_gen` is a better name here 😄

Reply:
In this case, `J` is actually used as a public key, the type is also `secp256k1_pubkey`